core/src/main/java/org/hibernate/ogm/util/impl/StringHelper.java

```diff
@@ -78,4 +79,31 @@ public static String join(Iterator<?> iterator, String separator) {
 		}
 		return "";
 	}
+
+	/**
+	 * Escape the following characters: ',",/,\
+	 * <p>
+	 * Example: ' becomes \'
+	 *
+	 * @param text the string to escape
+	 * @return the escaped text or {@code null} if text is {@code null}
+	 */
+	public static String escape(String text) {
```

Comment:
This is not a generic escaping utility so `StringHelper.escape()` is not a good choice.

So maybe I would keep it in the MongoDB module and at least call it `escapeJsonString()`.

By the way, are you sure we need to escape `/`? I find it a bit weird. I'm also wondering if we need to escape `"`, considering the string is wrapped into `'`. It might be the case, I'm not very familiar with MongoDB escaping rules but I think we should test it to be sure we are not escaping things too much. Unless it was already tested?